OtripleS.Web.Api.Tests.Unit/Services/StudentServiceTests/StudentServiceTests.cs

```diff
@@ -29,12 +30,26 @@ public StudentServiceTests()
         private Student CreateRandomStudent()
         {
             var filler = new Filler<Student>();
+
             filler.Setup()
                 .OnProperty(student => student.BirthDate).Use(this.dateTimeBroker.GetCurrentDateTime())
                 .OnProperty(student => student.CreatedDate).Use(this.dateTimeBroker.GetCurrentDateTime())
                 .OnProperty(student => student.UpdatedDate).Use(this.dateTimeBroker.GetCurrentDateTime());
 
             return filler.Create();
         }
+
+        private IQueryable<Student> CreateRandomStudents()
+        {
+            int randomNumber = new IntRange(min: 2, max: 10).GetValue();
+            var filler = new Filler<Student>();
+
+            filler.Setup()
+               .OnProperty(student => student.BirthDate).Use(this.dateTimeBroker.GetCurrentDateTime())
```

Comment:
Why would the `BirthDate` be a match for the current date time?
Instead just randomize the date - something like this:
```csharp
DateTimeOffset randomDate = new DateRange(earliestDate: new DateTime());
```

Reply:
```csharp
        private static DateTimeOffset GetRandomDateTime() =>
            new DateTimeRange(earliestDate: new DateTime()).GetValue();
```